ui/src/components/auth/Login.vue

```diff
@@ -72,19 +72,18 @@ function handleSubmit(): void {
           loggedInUser.avatar = Avatars[0];
         }
         userStore.updateCurrentUser(loggedInUser);
-        authStore.updateRegisteredUsername(null);
+        authStore.updateRegisteredUsername("");
         Swal.fire({
           icon: "success",
           title: "Success",
           text: "Login successful"
         }).then(() => {
           userStore.clearOptionalCookies();
-          if (previousAppUrl.value) {
-            window.location.href = previousAppUrl.value;
+          if (authReturnUrl.value) {
+            window.location.href = authReturnUrl.value;
           } else {
             router.push({ name: "LandingPage" });
           }
-          window.location.reload();
```

Comment:
i believe it was added back when the user theme, etc. were stored locally and logging in/out would cause problems with certain things persisting until a forced refresh was done, as far as i know it can be removed since all of that is stored in the user db now and its no longer an issue